changelog/2993.feature.rst

```diff
@@ -0,0 +1,3 @@
+The default location of the sunpy sample data has changed to be in the platform
```

Comment:
Suggest to just say "The default location of the sample data is now placed in platform specific data directory as provided by X".